### PR TITLE
AP_Relay:correct function param metadata

### PR DESCRIPTION
**Pull request by Hwurzburg**
NFC....except without this a user cant use these functions!!

### CODE REVIEW

**Review by IamPete1 — APPROVED** [review]
Thanks.

File: libraries/AP_Relay/AP_Relay_Params.cpp [hunk 1]
```diff
@@ -5,11 +5,11 @@ const AP_Param::GroupInfo AP_Relay_Params::var_info[] = {
     // @Param: FUNCTION
     // @DisplayName: Relay function
     // @Description: The function the relay channel is mapped to.
-    // @Values: 0:None
-    // @Values: 1:Relay
+    // @Values:{Copter, Rover, Plane, Blimp} 0:None
+    // @Values:{Copter, Rover, Plane, Blimp} 1:Relay
```
Comment (IamPete1) [inline libraries/AP_Relay/AP_Relay_Params.cpp:9]:
```suggestion
    // @Values{Copter, Rover, Plane, Blimp}: 0:None
    // @Values{Copter, Rover, Plane, Blimp}: 1:Relay
```

File: libraries/AP_Relay/AP_Relay_Params.cpp [hunk 2]
```diff
@@ -5,11 +5,11 @@ const AP_Param::GroupInfo AP_Relay_Params::var_info[] = {
...
     // @Values{Plane}: 2:Ignition
     // @Values{Plane, Copter}: 3:Parachute
-    // @Values: 4:Camera
+    // @Values:{Copter, Rover, Plane, Blimp} 4:Camera
```
Comment (IamPete1) [inline libraries/AP_Relay/AP_Relay_Params.cpp:12]:
```suggestion
    // @Values{Copter, Rover, Plane, Blimp}: 4:Camera
```

File: libraries/AP_Relay/AP_Relay_Params.cpp [hunk 3]
```diff
@@ -28,7 +28,7 @@ const AP_Param::GroupInfo AP_Relay_Params::var_info[] = {
 
     // @Param: DEFAULT
     // @DisplayName: Relay default state
-    // @Description: Should the relay default to on or off, this only applies to function Relay. All other uses will pick the appropriate default output state.
+    // @Description: Should the relay default to on or off, this only applies to RELAY_FUNC Relay (1). All other uses will pick the appropriate default output state from within the controlling function's parameters.
```
Comment (peterbarker) [inline libraries/AP_Relay/AP_Relay_Params.cpp:31]:
```suggestion
    // @Description: Should the relay default to on or off, this only applies to RELAYn_FUNCTION 1 ("Relay"). All other uses will pick the appropriate default output state from within the controlling function's parameters.
```

File: libraries/AP_Relay/AP_Relay_Params.cpp [hunk 4]
```diff
@@ -5,11 +5,11 @@ const AP_Param::GroupInfo AP_Relay_Params::var_info[] = {
     // @Param: FUNCTION
     // @DisplayName: Relay function
     // @Description: The function the relay channel is mapped to.
-    // @Values: 0:None
-    // @Values: 1:Relay
+    // @Values{Copter, Rover, Plane, Blimp}: 0:None
```
Comment (peterbarker) [inline libraries/AP_Relay/AP_Relay_Params.cpp:8]:
```suggestion
    // @Values{Copter, Rover, Plane, Blimp, Sub}: 0:None
```

File: libraries/AP_Relay/AP_Relay_Params.cpp [hunk 5]
```diff
@@ -5,11 +5,11 @@ const AP_Param::GroupInfo AP_Relay_Params::var_info[] = {
...
+    // @Values{Copter, Rover, Plane, Blimp}: 1:Relay
```
Comment (peterbarker) [inline libraries/AP_Relay/AP_Relay_Params.cpp:9]:
```suggestion
    // @Values{Copter, Rover, Plane, Blimp, Sub}: 1:Relay
```

File: libraries/AP_Relay/AP_Relay_Params.cpp [hunk 6]
```diff
@@ -5,11 +5,11 @@ const AP_Param::GroupInfo AP_Relay_Params::var_info[] = {
...
     // @Values{Plane}: 2:Ignition
     // @Values{Plane, Copter}: 3:Parachute
-    // @Values: 4:Camera
+    // @Values{Copter, Rover, Plane, Blimp}: 4:Camera
```
Comment (peterbarker) [inline libraries/AP_Relay/AP_Relay_Params.cpp:12]:
```suggestion
    // @Values{Copter, Rover, Plane, Blimp, Sub}: 4:Camera
```

**Review by peterbarker — CHANGES_REQUESTED** [review]

File: libraries/AP_Relay/AP_Relay_Params.cpp (hunk 3 above)
Comment (Hwurzburg) [reply libraries/AP_Relay/AP_Relay_Params.cpp:31]: done

File: libraries/AP_Relay/AP_Relay_Params.cpp (hunk 4 above)
Comment (Hwurzburg) [reply libraries/AP_Relay/AP_Relay_Params.cpp]: done

File: libraries/AP_Relay/AP_Relay_Params.cpp (hunk 6 above)
Comment (Hwurzburg) [reply libraries/AP_Relay/AP_Relay_Params.cpp]: done

File: libraries/AP_Relay/AP_Relay_Params.cpp (hunk 5 above)
Comment (Hwurzburg) [reply libraries/AP_Relay/AP_Relay_Params.cpp]: done

**Comment by magicrub** [comment]
Is this ready to merge? Seems OK to me now..

**Comment by Hwurzburg** [comment]
I think so....

**Review by magicrub — APPROVED** [review]

**Comment by Hwurzburg** [comment]
thanks!

**Pull request merged**